res/css/views/settings/tabs/user/_HelpUserSettingsTab.pcss

```diff
@@ -15,6 +15,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+/*
+     overflow-x and overflow-y is set to scrollable for 
+     .max_CopyableText class under div tag which is for Access Key
+     field, So it won't go outside boundary.
+*/
```

Comment:
why is this way up here, so far from the settings in question?

Reply:
Hello @richvdh, I thought it would be better to mention with before comment, But I made comment just above the code logic now. Thanks!